client/tests/unit/specs/testCryptpad.spec.ts

```diff
@@ -10,6 +11,20 @@ vi.mock('@/services/environment', () => ({
     isEditicsEnabled: vi.fn(),
   },
 }));
+
+vi.mock('@/services/informationManager', () => ({
```

Comment:
I wouldn't touch this file with a ten-foot pole. Just remove the information manager from the Cryptpad class.
Those tests should be e2e tests instead.